Code/GraphMol/SmilesParse/SmilesParse.h

```diff
@@ -32,11 +104,46 @@ struct RDKIT_SMILESPARSE_EXPORT SmilesParserParams {
   bool skipCleanup =
       false; /**<  skip the final cleanup stage (for internal use) */
 };
-RDKIT_SMILESPARSE_EXPORT RWMol *SmilesToMol(const std::string &smi,
-                                            const SmilesParserParams &params);
 
-RDKIT_SMILESPARSE_EXPORT Atom *SmilesToAtom(const std::string &smi);
-RDKIT_SMILESPARSE_EXPORT Bond *SmilesToBond(const std::string &smi);
+struct RDKIT_SMILESPARSE_EXPORT SmartsParserParams {
+  int debugParse = 0; /**< enable debugging in the SMARTS parser*/
+  std::map<std::string, std::string> *replacements =
+      nullptr;               /**< allows SMARTS "macros" */
+  bool allowCXSMILES = true; /**< recognize and parse CXSMILES extensions */
+  bool strictCXSMILES =
+      true; /**< throw an exception if the CXSMILES parsing fails */
+  bool parseName = true; /**< parse (and set) the molecule name as well */
+  bool mergeHs =
+      false; /**< toggles merging H atoms in the SMARTS into neighboring atoms*/
+  bool skipCleanup =
+      false; /**<  skip the final cleanup stage (for internal use) */
+};
+
+inline RDKit::RWMol *SmilesToMol(const std::string &smi,
```

Comment:
In Python, we use `MolFromSmiles()`. In C++, it's `SmilesToMol()`.

As someone who bounces back and forth between the languages frequently, it would be nice if they matched. I looked in github, and, like, every hit for `SmilesToMol()` is in a repo owned by someone I know (or is a CDK reference). There are 10x more `MolFromSmiles()`.

Should we take the v2 opportunity to change the C++ function name and signature to match the Python one?

Reply:
> Should we take the v2 opportunity to change the C++ function name and signature to match the Python one?

I really like this idea.